internal/serverinstall/nomad.go

```diff
@@ -1394,11 +1428,11 @@ func (i *NomadInstaller) UpgradeFlags(set *flag.Set) {
 		Usage:  "Nomad host volume name.",
 	})
 
-	set.BoolVar(&flag.BoolVar{
-		Name:    "nomad-consul-service",
-		Target:  &i.config.consulService,
+	set.StringVar(&flag.StringVar{
+		Name:    "nomad-service-provider",
```

Comment:
Might be a good idea to support the old flag for a release or two rather than remove it immediately.

Reply:
@evanphx updated!